config/config.go

```diff
@@ -1,3 +1,16 @@
+// package config provides the ability to load and save config.json and
```

Comment:
`P` should be capitalized as per golang doc convension.
```suggestion
// Package config provides the ability to load and save config.json and
```